<?xml version="1.0" ?>
<launch>
<!--
  <node pkg="falkor_drivers" name="i2c_driver" type="i2c_driver.py" />
-->
  <node pkg="falkor_drivers" name="i2c_driver" type="i2c_driver.py" />
  <node type="imu_filter_node" pkg="imu_filter_madgwick" name="imu_filter_node">
    <param name="fixed_frame" value="/map/magnet" />
    <remap to="raw_imu" from="imu/data_raw" />
    <remap to="magnetic" from="imu/mag" />
  </node>
  <node pkg="falkor_drivers" name="magnetometer_calibrator"
	type="magnetometer_calibrator.py" output="screen"/>
</launch>
